Makefile

```diff
@@ -3,35 +3,47 @@
 # export CROSS_COMPILE=arm-none-linux-gnueabi-
 # make
 
+EXENAME = mcrcon
+PREFIX ?= /usr/local
+
+CFLAGS ?= -std=gnu99 -Wall -Wextra -Wpedantic -Os -s
+EXTRAFLAGS ?= -fstack-protector-strong
+
+CC ?= gcc
```

Comment:
I think we can safely drop the override of `CC`, as (I think?) it's guaranteed to be present, and already respects fetching the value from the env, afaict.